internal/featuretests/httpproxy.go

```diff
@@ -16,11 +16,45 @@ package featuretests
 // HTTPProxy helpers
 
 import (
+	"strings"
+
 	projcontour "github.com/projectcontour/contour/apis/projectcontour/v1"
 )
 
-func prefixCondition(prefix string) []projcontour.Condition {
+// Returns an array of Conditions
+func prefixConditions(prefix string) []projcontour.Condition {
```

Comment:
I don't think this is accurate, the method is called prefixConditions plural, but only takes a single value.

Reply:
It's based on what it returns. I need a way to pass a set of conditions which is why the `conditions()` method was added. I can add that everywhere to resolve.

Reply:
> It's based on what it returns. 

Then it should probably be something like
```
func conditions(first projcontour.Condition, rest ...projcontour.Condition) []projcontour.Condition {
   return append([]projcontour.Condition{first}, rest...)
}
```
I this would also make `prefixCondition` more annoying to use as it would become
```
Conditions: conditions(prefixCondition("/yada")),
```
Calling `prefixCondition` that was probably a mistake, but I was thinking about it from the POV that the name described what it took, not what it returned